gstreamer/classify.py

```diff
@@ -35,6 +39,23 @@ def generate_svg(size, text_lines):
       dwg.add(dwg.text(line, insert=(10, y*20), fill='white', font_size='20'))
     return dwg.tostring()
 
+def output_tensor(interpreter):
+    """Returns dequantized output tensor."""
+    output_details = interpreter.get_output_details()[0]
+    output_data = np.squeeze(interpreter.tensor(output_details['index'])())
+    scale, zero_point = output_details['quantization']
+    return scale * (output_data - zero_point)
+
+def get_output(interpreter, top_k, score_threshold):
+    """Returns no more than top_k classes with score >= score_threshold."""
+    scores = output_tensor(interpreter)
+    classes = [
+        Class(i, scores[i])
+        for i in np.argpartition(scores, -top_k)[-top_k:]
+        if scores[i] >= score_threshold
+    ]
+    return sorted(classes, key=operator.itemgetter(1), reverse=True)
```

Comment:
usually i see key funcs as lambda's, e.g. `key lambda i: i[1]`.  Is using the `operator` module a local idiom?

Reply:
For get_output I referenced [this TFLite example ](https://github.com/google-coral/tflite/blob/master/python/examples/classification/classify.py#L49) and they use the `operator` module here. Thoughts?

Reply:
a quick google suggests its slightly faster! https://stackoverflow.com/questions/17243620/operator-itemgetter-or-lambda

Reply:
+1 to operator.itemgetter() though it really doesn't matter in practice here.